playbooks/roles/edxapp/templates/edx/app/edxapp/worker.sh.j2

```diff
@@ -4,19 +4,31 @@
 
 {% set edxapp_venv_bin = edxapp_venv_dir + "/bin" %}
 source {{ edxapp_app_dir }}/edxapp_env
+{% set executable = edxapp_venv_bin + '/celery' %}
+
 {% if COMMON_ENABLE_NEWRELIC_APP %}
-{% set executable = edxapp_venv_bin + '/newrelic-admin run-program ' + edxapp_venv_bin + '/celery' %}
+{% set executable = edxapp_venv_bin + '/newrelic-admin run-program ' + executable %}
 
 export NEW_RELIC_CONFIG_FILE="{{ edxapp_app_dir }}/newrelic.ini"
 if command -v ec2metadata >/dev/null 2>&1; then
   INSTANCEID=$(ec2metadata --instance-id);
   HOSTNAME=$(hostname)
   export NEW_RELIC_PROCESS_HOST_DISPLAY_NAME="$HOSTNAME-$INSTANCEID"
 fi
-{% else %}
-{% set executable = edxapp_venv_bin + '/celery' %}
 {% endif %}
 
+{% if EDXAPP_DATADOG_ENABLE %}
+{% set executable = edxapp_venv_bin + '/ddtrace-run ' + executable %}
+export DD_TAGS=("service:edx-edxapp-{{ SERVICE_VARIANT }}-workers", "queue:{{EDX_REST_API_CLIENT_NAME}}")
```

Comment:
1. I don't think array syntax will work here. I believe what we want is this:
    ```suggestion
    export DD_TAGS="service:edx-edxapp-{{ SERVICE_VARIANT }}-workers,queue:{{EDX_REST_API_CLIENT_NAME}}"
    ```
2. Do we want `edx-edxapp-VARIANT-workers`, or `edx-edxapp-workers-VARIANT`? The latter would be a better match for what we have in New Relic.
3. Rather than introducing a secondary use for `EDX_REST_API_CLIENT_NAME` can we set an environment variable in the workers.conf like `EDXAPP_WORKERS_QUEUE`? Otherwise we end up yoking those two features together.